recipes/gpi-framework/yum_requirements.txt

```diff
@@ -0,0 +1,4 @@
++ mesa-libGL-devel
++ libSM
++ libXrender
```

Comment:
libsm and libxrender have conda-forge packages, so please do not use the yum versions

Reply:
Thanks Scott, believe this is addressed by the latest commit.